keyboards/smolpad/keyboard.json

```
@@ -0,0 +1,56 @@
+{
+    "manufacturer": "wolf-yuan",
+    "keyboard_name": "smolpad",
+    "maintainer": "wolf-yuan-6115",
+    "bootloader": "rp2040",
+    "diode_direction": "COL2ROW",
+    "encoder": {
+        "rotary": [
+            {"pin_a": "GP29", "pin_b": "GP0"}
+        ]
+    },
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "encoder": true,
+        "extrakey": true,
+        "mousekey": false,
+        "nkro": true,
```

Note, if OLEDs are to be enabled at the keyboard level, there needs to exist some configuration for them at the keyboard level in a `smolpad.c` file. My suggestion would be to move the existing OLED code out of the `default` keymap, and to the keyboard level.

```suggestion
        "nkro": true,
        "oled": true,
```